packages/qwik/src/core/util/markers.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
```

Comment:
I don't think this file belongs here. Was it included by accident?

Reply:
probably auto generated, yeah. sorry